stackslib/src/cli.rs

```diff
@@ -68,6 +69,44 @@ impl StacksChainConfig {
             epochs: STACKS_EPOCHS_MAINNET.to_vec(),
         }
     }
+
+    pub fn default_testnet() -> Self {
+        let mut pox_constants = PoxConstants::regtest_default();
+        pox_constants.prepare_length = 100;
+        pox_constants.reward_cycle_length = 900;
+        pox_constants.v1_unlock_height = 3;
+        pox_constants.v2_unlock_height = 5;
+        pox_constants.pox_3_activation_height = 5;
+        pox_constants.pox_4_activation_height = 6;
+        pox_constants.v3_unlock_height = 7;
+        let mut epochs = STACKS_EPOCHS_REGTEST.to_vec();
+        epochs[0].start_height = 0;
```

Comment:
At some point, I had started making a change that added an `index()` to `StacksEpochId`. I couldn't decide if I liked it or not... what do you think? Would that be helpful for places like this? So for example, here you could write:
```rust
epochs[StacksEpochId::Epoch20.index()].start_height = 0;
```